test/lib_test.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+NC='\033[0m'
+CYAN='\033[0;36m'
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+
+INPUT_FILES="lib/*.ods"
+TMP_FILE=$(mktemp "compiled.XXXXX")
+printf "${CYAN}Running compiler 'pass' tests...\n${NC}"
+
+for input_file in $INPUT_FILES; do
+    output_file=${input_file/.ods/.out}
+
+    # compile odds program to temp python file
+    ../compiler/odds.sh -c $input_file $TMP_FILE
```

Comment:
we want to make sure standard library functions are working, so we are only comparing the output file and not the python file.